charts/artifact-hub/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: artifact-hub
 description: Artifact Hub is a web-based application that enables finding, installing, and publishing Kubernetes packages.
 type: application
-version: 0.13.1
-appVersion: 0.13.0
+version: 0.13.2
+appVersion: 0.13.1
```

Comment:
We usually don't bump the app version in the chart for every change we make to the code, only when we cut a release.

Reply:
I know, but this change depends on the changes in PR #1043. Therefore I would have waited with a merge until a new release has been created.
I mean it would technically work with the default values, but the schema / values.yaml now implies that one can set the config and cache directory even though that won't work with 0.13.0.

Should I keep it that way (and wait for this PR for a release containing #1043), or should I revert the appVersion bump?

Reply:
I see what you mean. This happens often when we add new features until they become part of the new release. But it's not usually a problem because they won't be either visible in the [docs](https://artifacthub.io/packages/helm/artifact-hub/artifact-hub?modal=values-schema) or in any chart version published until a new release is cut. If users were relying on the values schema available in the repo, they might be out of sync. But I'd say that's expected.

That's why we don't usually bump this version, as it doesn't really exist. In a few days when `0.14.0` is published I'll bump it, leave it with me. Thanks!

Reply:
Done (reverted appVersion to 0.13.0)